src/components/EntityPage/EntityTitle.js

```diff
@@ -45,7 +45,7 @@ const EntityTitle = ({ icon, title, tags = [] }) => (
 EntityTitle.propTypes = {
   title: PropTypes.string,
   icon: PropTypes.string,
-  tags: PropTypes.arrayOf(PropTypes.string),
+  tags: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.string), PropTypes.array]),
```

Comment:
That's equivalent to `tags: PropTypes.array`

Reply:
corrected